enter

```diff
@@ -1,2 +1,8 @@
 #!/bin/sh
-docker run -it -v "$(pwd):/app" krig/godev /bin/bash
+destdir="/root/go/src/github.com/ClusterLabs/hawk-apiserver"
+common_opts="-t --mount type=bind,source=$(pwd),destination=$destdir"
+if [ "$1" = "--ci" ]; then
+	docker run $common_opts krig/godev /bin/sh -c "systemctl start dbus; cd $destdir; ./ci.sh"
```

Comment:
@krig I think we should move it to shap organization. You should be able to push it there already.

Reply:
ah yes, I wanted to do that.